### PR TITLE
org.adi.Scopy.json: Update the dependencies in order to build Scopy using GR 3.8

### PR DESCRIPTION
**Pull request by AlexandraTrifan**
- gr-iio is also updated to the corresponding 3.8 version.
- libm2k, gr-m2k and gr-scopy are added as dependencies.
- update to Qt 5.12.

Signed-off-by: Alexandra.Trifan <Alexandra.Trifan@analog.com>

### CODE REVIEW

File: org.adi.Scopy.json [hunk 1]
```diff
@@ -441,11 +468,12 @@
 			"name": "scopy",
 			"builddir": true,
 			"buildsystem": "cmake",
-			"config-opts": [ "-DCMAKE_INSTALL_PREFIX:PATH=/app", "-DWITH_DOC=OFF", "-DCMAKE_BUILD_TYPE=Release", "-DBREAKPAD_HANDLER=OFF" ],
+			"config-opts": [ "-DCMAKE_INSTALL_PREFIX:PATH=/app", "-DCMAKE_PREFIX_PATH=/app/lib/pkgconfig;/app/lib/cmake", "-DWITH_DOC=OFF", "-DCMAKE_BUILD_TYPE=Release", "-DBREAKPAD_HANDLER=OFF" ],
 			"sources": [
 				{
 					"type": "git",
-					"url": "https://github.com/analogdevicesinc/scopy"
+					"url": "https://github.com/adisuciu/scopy",
```
Comment (adisuciu) [inline org.adi.Scopy.json]: You can use analogdevicesinc mirror here

File: org.adi.Scopy.json [hunk 2]
```diff
@@ -441,11 +468,12 @@
...
+					"branch": "ubuntu_appveyor"
```
Comment (adisuciu) [inline org.adi.Scopy.json]: Can this be built for all branches ? Get scopy branch from parameter ?

File: org.adi.Scopy.json (hunk 1 above)
Comment (AlexandraTrifan) [reply org.adi.Scopy.json]: Done

File: org.adi.Scopy.json (hunk 2 above)
Comment (AlexandraTrifan) [reply org.adi.Scopy.json]: If we build this locally, we can put what branch we need. 
In the Scopy Appveyor, this is modified to be the exact same branch that the Appveyor build uses.

**Pull request merged**